### PR TITLE
Tweaks

### PR DESCRIPTION
**Pull request by justyna124**

### CODE REVIEW

File: code/config.js [hunk 1]
```diff
@@ -5,3 +5,6 @@ export const MANAGER_APP_ANDROID_URL = 'https://play.google.com/store/apps/detai
 export const MANAGER_APP_IOS_URL = 'https://play.google.com/store/apps/details?id=fun.scan.manager';
 export const DISTRIBUTOR_APP_ANDROID_URL = 'https://play.google.com/store/apps/details?id=fun.scan.distributor';
 export const DISTRIBUTOR_APP_IOS_URL = 'https://play.google.com/store/apps/details?id=fun.scan.distributor';
+
+export class MANAGER_APP_ANDROID_URL_ {
```
Comment (blabno) [inline code/config.js]: Why this change?

File: code/downloadMobileApp.js [hunk 2]
```diff
@@ -10,10 +10,10 @@ const DownloadMobileApp = ({ _relativeURL, androidUrl, iosUrl }) => {
       <div className={'download-mobile-app'}>
         <div>Pobierz aplikację mobilną:</div>
         <a href={androidUrl}>
-          <img src={_relativeURL('/assets/images/google-play.png')} alt={'Pobierz aplikację z Google Play'}/>
+          <img src={_relativeURL('../../../../assets/images/google-play.png')} alt={'Pobierz aplikację z Google Play'}/>
```
Comment (blabno) [inline code/downloadMobileApp.js]: Please use absolute urls, the `_relativeURL` function does the job of making them relative.

File: content/dokumentacja/body.md [hunk 3]
```diff
@@ -5,3 +5,5 @@ layout: row
 # Dokumentacja
 
 * [ScanFun Manager](/dokumentacja/scan-fun-manager)
+* [ScanFun Kasjer](/dokumentacja/scan-fun-clerk)
+* [ScanFun Kolekcjoner](/dokumentacja/scan-fun-collector)
```
Comment (blabno) [inline content/dokumentacja/body.md]: `ScanFun Kolekcjoner` => `ScanFun`

File: content/dokumentacja/body.md [hunk 4]
```diff
@@ -5,3 +5,5 @@ layout: row
 # Dokumentacja
 
 * [ScanFun Manager](/dokumentacja/scan-fun-manager)
+* [ScanFun Kasjer](/dokumentacja/scan-fun-clerk)
```
Comment (blabno) [inline content/dokumentacja/body.md]: There is no such app as `scan-fun-clerk`

**Review by blabno — CHANGES_REQUESTED** [review]

File: content/dokumentacja/body.md (hunk 3 above)
Comment (blabno) [reply content/dokumentacja/body.md]: Those links do not lead anywhere so do not add them yet.

File: content/dokumentacja/scan-fun-manager/index.yml [hunk 5]
```diff
@@ -3,5 +3,6 @@ header:
   - /_shared/header.md
 main:
   - body.md
+  - download-manager-mobile-app.md
```
Comment (blabno) [inline content/dokumentacja/scan-fun-manager/index.yml]: No such file

File: content/dokumentacja/scan-fun-manager/rejestracja/index.yml [hunk 6]
```diff
@@ -3,5 +3,11 @@ header:
   - /_shared/header.md
 main:
   - body.md
+  - registration-first-step.md
```
Comment (blabno) [inline content/dokumentacja/scan-fun-manager/rejestracja/index.yml]: Those files are missing

**Review by blabno — CHANGES_REQUESTED** [review]

**Comment by justyna124** [comment]
In code/config.js
<https://github.com/blabno/scan-fun-website/pull/1#discussion_r362719805>:

> @@ -5,3 +5,6 @@ export const MANAGER_APP_ANDROID_URL = 'https://play.google.com/store/apps/detai

 export const MANAGER_APP_IOS_URL =
'https://play.google.com/store/apps/details?id=fun.scan.manager';

 export const DISTRIBUTOR_APP_ANDROID_URL =
'https://play.google.com/store/apps/details?id=fun.scan.distributor';

 export const DISTRIBUTOR_APP_IOS_URL =
'https://play.google.com/store/apps/details?id=fun.scan.distributor';

+

+export class MANAGER_APP_ANDROID_URL_ {


Why this change? - Chciałam dodać przyciski do sklepu żeby pobrać aplikację
scan fun manager, ale nie udało mi się wyświetlić zdjęć


In code/downloadMobileApp.js
<https://github.com/blabno/scan-fun-website/pull/1#discussion_r362719866>:

> @@ -10,10 +10,10 @@ const DownloadMobileApp = ({ _relativeURL, androidUrl, iosUrl }) => {

       <div className={'download-mobile-app'}>

         <div>Pobierz aplikację mobilną:</div>

         <a href={androidUrl}>

-          

+          


Please use absolute urls, the _relativeURL function does the job of making
them relative. - Poprawiłam

ScanFun Kolekcjoner => ScanFun - Poprawiłam

There is no such app as scan-fun-clerk - poprawiła na scan fun portal

Brakujace pliki doślę.




pt., 3 sty 2020 o 08:06 Bernard Labno <notifications@github.com> napisał(a):
